docs/azure/index-old.yml

```diff
@@ -0,0 +1,201 @@
+### YamlMime:Hub
+
+title: Azure for .NET developers
+summary: Find the information you need to use .NET on Azure
+brand: dotnet
+
+metadata:
+  title: Azure for .NET developers
+  description: Samples, tutorials, and education for .NET on Azure
+  ms.topic: hub-page
+  ms.prod: azure-dotnet
+  ms.date: 03/04/2020
+
+highlightedContent:
+# itemType: architecture | concept | deploy | download | get-started | how-to-guide | learn | overview | quickstart | reference | tutorial | whats-new 
+  items:
+    - itemType: quickstart
+      title: Deploy an ASP.NET web app
+      url: https://docs.microsoft.com/azure/app-service/app-service-web-get-started-dotnet
+    - itemType: quickstart
+      title: Build a serverless function
+      url: https://docs.microsoft.com/azure/azure-functions/functions-create-your-first-function-visual-studio
+    - itemType: quickstart
+      title: ASP.NET Core and Docker
+      url: https://docs.microsoft.com/visualstudio/containers/container-tools?view=vs-2019
+    - itemType: learn
+      title: Azure fundamentals
+      url: https://docs.microsoft.com/learn/paths/azure-fundamentals/
+    - itemType: tutorial
+      title: Deploy a .NET app with Azure DevOps
+      url: https://docs.microsoft.com/en-us/azure/devops/pipelines/ecosystems/dotnet-core?view=azure-devops
+    - itemType: tutorial
+      title: Full stack authentication end-to-end in App Service
+      url: https://docs.microsoft.com/azure/app-service/app-service-web-tutorial-auth-aad
+    - itemType: learn
+      title: Secure custom APIs with Microsoft Identity
+      url: https://docs.microsoft.com/learn/modules/identity-secure-custom-api/
+    - itemType: download
+      title: Sample code
```

Comment:
I find the term "sample code" confusing. To me "sample code" is singular, i.e. code of one app. This link points to samples and I would just call it "samples".

Reply:
You're probably right. "Sample code" with the word "code" was a request straight from ScottGu a couple years ago but I can probably drop that now.